.github/workflows/ci_llama_3.1_405b_fp16_tp8.yml

```diff
@@ -4,13 +4,13 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-name: Llama 3.1 405 FP16 TP8 Benchmarking Tests
+name: Release: Llama 3.1 405 FP16 TP8 Benchmarking Tests
 
 on:
   workflow_dispatch:
   schedule:
     # Weekdays at 11:00 AM UTC = 03:00 AM PST / 04:00 AM PDT
-    - cron: "0 11 * * 0-6"
+    - cron: "0 9 * * 7"
```

Comment:
Drive-by: If you update the schedule it would be helpful to update the comments that elaborates on the interval as well.